airflow/www/static/js/dag/details/taskInstance/Logs/utils.ts

```diff
@@ -97,6 +101,17 @@ export const parseLogs = (
         line.includes(fileSourceFilter)
       )
     ) {
+      const tagRegex = /<[a-z][\s\S]*>/i;
```

Comment:
Perhaps we should use a real library to sanitize our string: https://www.npmjs.com/package/sanitize-html

Reply:
Yeah makes sense, working on it